pytest/empty-contract-rs/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "empty-contract-rs"
+version = "0.1.0"
+authors = ["Near Inc <hello@nearprotocol.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { git = "https://github.com/nearprotocol/json", rev = "1f5779f3b0bd3d2a4b0b975abc46f3d3fe873331", features = ["no_floats"] }
```

Comment:
We can now use official serde_json. No need to depend on our fork.

Reply:
Nice